next.config.js

```diff
@@ -356,6 +356,11 @@ module.exports = () => {
           destination: '/docs/instrumentation/opentelemetry-springboot/',
           permanent: true,
         },
+        {
```

Comment:
We can skip this redirect and instead name the mdx file `id` as `opentelemetry-quarkus`